app/src/test/java/protect/card_locker/DatabaseTest.java

```diff
@@ -317,44 +317,44 @@ public void updateGroup() {
         assertEquals(1, DBHelper.getGroupCount(mDatabase));
 
         // Add card to group
-        Group group = DBHelper.getGroup(mDatabase, "group one");
+        Group group = DBHelper.getGroupByName(mDatabase, "group one");
```

Comment:
We should also test if we can get a group by ID, as that's the actual canonical thing.